app/components/elements/forms/base_input_collection_component.rb

```diff
@@ -5,21 +5,26 @@ module Forms
     # Base component for a set of inputs for a collection, e.g. checkboxes or radio buttons
     class BaseInputCollectionComponent < ApplicationComponent
       def initialize(form:, field_name:, input_collection:, value_method: :to_s, text_method: :to_s, # rubocop:disable Metrics/ParameterLists
-                     div_classes: [])
+                     div_classes: [], input_data: {})
```

Comment:
Perhaps nitpicky and not a blocker, would it make sense to move the `rubocop:disable` onto this line or is there a good reason for it to stay on 7?